src/block.cpp

```diff
@@ -28,6 +28,8 @@
 #error Must build without C++ exceptions
 #endif
 
+#define _Bool bool
```

Comment:
This looks like it might cause problems for other builds

Reply:
Indeed. This shouldn't even be there!